From 340642b5e6d89b1b04c8bed04583636bae78d017 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@gmail.com>
Date: Thu, 23 May 2024 18:27:50 +0200
Subject: [PATCH] webOS.cmake: update cmake_minimum_required to 3.5.0

:Release Notes:
To avoid warning in log.do_install.

:Detailed Notes:
CMake Deprecation Warning at CMakeLists.txt:5 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:

Change-Id: I6b7df9c313b8d8cbc7a0070694bea86765c71633
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/cmake-modules-webos/+/408140 webOS.cmake: update cmake_minimum_required to 3.5.0]

 CMakeLists.txt    | 2 +-
 webOS/webOS.cmake | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8713663..7697757 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@
 
 # Commented-out lines are used for testing
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5.0)
 
 project(cmake-modules-webos NONE)
 
diff --git a/webOS/webOS.cmake b/webOS/webOS.cmake
index 3d48603..c1dac04 100644
--- a/webOS/webOS.cmake
+++ b/webOS/webOS.cmake
@@ -58,7 +58,7 @@
 
 # TODO: Using PARENT_SCOPE, see if any macros can be turned into functions
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5.0)
 
 include(CMakeParseArguments)
 
